src/utils.js

```diff
@@ -13,3 +13,7 @@ module.exports.showDataWithDisplay = (data, display) => {
     console.log(JSON.stringify(data, null, 2))
   }
 }
+
+module.exports.add0x = hex => {
+  return hex.indexOf('0x') === 0 ? hex : `0x${hex}`
+}
```

Comment:
From: https://github.com/protofire/eth-cli/pull/28/commits/87bf19eed0b93f3f18d7c899f008dfbf23a059a0#diff-4c1f7037bae28dfe3f629ce1533375f0L20

```javascript
if (address[0] === '0' && (address[1] === 'x' || address[1] === 'X')) {
```

It looks like the `X` (uppercase X) case isn't covered after this refactor

Reply:
ohhf! I missed this comment: https://github.com/protofire/eth-cli/pull/28#discussion_r220335819

> (I could've make add0x case insensitive, but I don't think it's worth the extra complexity.)